stories/SideNav.stories.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import SideNav from '../src/SideNav';
+import SideNavItem from '../src/SideNavItem';
+import Button from '../src/Button';
+
+const stories = storiesOf('Javascript|SideNav', module);
+const reactMaterializeLogo = require('./react-materialize-logo.svg');
+
+stories.addParameters({
+  info: {
+    text: `This is a slide out menu.`
+  }
+});
+
+stories.add('Default', () => (
+  <div>
+    <style>
+      {`
```

Comment:
@alextrastero, some sugestions here? I did it this way because of a div with z-index:0 generated by storybook. That z-index:0 wasn't allowing SideNav to work properly on its story.

Reply:
oh, not sure, I don't think it really matters if the storybook source doesn't look 💯